build/build.js

```diff
@@ -414,6 +414,9 @@ function genericOktaConfiguration() {
   config.AUTH_REQUEST.response_type = 'code';
   config.AUTH_REQUEST.scope = '${scope}';
   config.AUTH_REQUEST.redirect_uri = 'https://${domain-name}${callback-path}';
+  if ('${idp}') {
```

Comment:
This line is redundant and should be removed. Note that this function runs at build time (not run time) and just generates a templated config file.

Reply:
Updated.